rstudio/rstudio
# feat(i18n): enable fr locale support

### Intent

Enables use of the fr locale. Also fixes up some incorrect messages and removes some unused resource bundle keys.

### Approach

Enables use of the fr locale by adding it to the list of supported locales declared in `RStudio.gwt.xml`.

### QA Notes

I was able to verify in Desktop - you can view in desktop mode by uncommenting out the set-property for locale element in `RStudioDesktopSuperDevMode.gwt.xml`:

![image](https://user-images.githubusercontent.com/14057112/145639990-6d68cff3-bdd3-44a3-9eed-0c56035fb8b6.png)

French values are visible in the Global Options.

For web, you should be able to open RStudio with french locale by setting the url to http://localhost:8787/?locale=fr.

### Checklist


cc @gtritchie
